### PR TITLE
Fix section 4 for 2019

### PR DESCRIPTION
**Pull request by odarbelaeze**
Hi,

Now most of section 4 runs. Tomorrow I will fix the last part about troubleshooting. The key points I'm missing there are:

- `logshow` moved somewhere else in the cli and I couldn't find where today.
- `calcjob.res.warnings`, I don't think it exist anymore, so you get the logs and translate the error code into a message.

### CODE REVIEW

**Comment by ltalirz** [comment]
@odarbelaeze thanks a lot for your help!

@sphuber Could you clarify the two points oscar mentioned above?

File: docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.md [hunk 1]
```diff
@@ -32,71 +32,68 @@ verdi daemon start
...
-This returns a builder that helps in setting up the inputs for the `PwCalculation` class (associated to the `quantumespresso.pw` plugin, i.e. the default plugin for the code you chose before).
+This returns a builder that helps in setting up the inputs for the `PwCalculation` class (associated to the `quantumespresso.pw` plugin, the default plugin for the code you chose before).
 
 As the first step, you can assign a (short) label or a (long) description to the calculation that you are going to create, that you might find convenient in the future. This can be achieved with:
 
 ``` python
- builder.label = "PW test"
- builder.description = "My first AiiDA calc with Quantum ESPRESSO on BaTiO3"
+builder.metadata.label = "PW test"
+builder.metadata.description = "My first AiiDA calc with Quantum ESPRESSO on Si"
```
Comment (ltalirz) [inline docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.md:76]: just wondering whether the calculation they are doing is on Si or BaTiO3 here...

Comment (odarbelaeze) [reply docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.md:76]: I'll double check tomorrow, I'm not sure myself.

**Comment by sphuber** [comment]
`logshow` is now `verdi process report`.
The `node.res` shortcut should still work and I think the current parser still returns a key `warnings` in the `output_parameters` although we should check this.

File: docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.md (hunk 1 above)
Comment (odarbelaeze) [reply docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.md:76]: Indeed, at that point users are working with an `Si` structure.

**Comment by odarbelaeze** [comment]
We might want to remove the warnings part, from there, since it's an implementation detail of the quantum espresso plugin rather than something that the users will encounter in every plugin.

Does that sound like a good plan?

File: docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.md [hunk 2]
```diff
@@ -223,71 +225,77 @@ By now, it is possible that the calculation you submitted has already finished,
...
 ```console
- verdi calculation outputcat <pk_number> | less
+verdi calcjob outputcat <pk_number> | less
 ```
 
 This can be a good primer for problem inspection. For something more compact, you can also try to inspect the calculation log (from AiiDA):
 
 ```console
- verdi calculation logshow <pk_number>
+verdi cacljob logshow <pk_number>
```
Comment (ltalirz) [inline docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.md]:
```suggestion
verdi calcjob logshow <pk_number>
```

**Comment by ltalirz** [comment]
> We might want to remove the warnings part, from there, since it's an implementation detail of the quantum espresso plugin rather than something that the users will encounter in every plugin.

Makes sense to me

**Comment by odarbelaeze** [comment]
I think this is complete now, mind you that for the `calculation.res` parts we would need at least `tag:8fb1a75c9df4b175197db5259f41e97ff47bfeea`

https://github.com/aiidateam/aiida-quantumespresso/tree/8fb1a75c9df4b175197db5259f41e97ff47bfeea

**Comment by odarbelaeze** [comment]
Reverting those off-topic commits.

**Comment by ltalirz** [comment]
@odarbelaeze do you understand why I cannot "update" your branch here?
Also it seems strange there are no conflicts with the base branch since I edited the .travis.yml

**Comment by ltalirz** [comment]
Ah, I see you removed the commits

**Review by ltalirz — APPROVED** [review]
feel free to merge if you're done with the changes

**Pull request merged**